test/sqllogictest/explain-pushdown.slt

```diff
@@ -30,15 +30,37 @@ EOF
 query error db error: ERROR: EXPLAIN FILTER PUSHDOWN is not supported
 EXPLAIN FILTER PUSHDOWN FOR SELECT * FROM numbers where value > 10;
 
-# Even when the feature flag is enabled, the feature is blocked in adapter
+# Even when the feature flag is enabled, the feature is blocked in adapter for most queries
 
 simple conn=mz_system,user=mz_system
 ALTER SYSTEM SET enable_explain_pushdown = true
 ----
 COMPLETE 0
 
-query error db error: ERROR: EXPLAIN FILTER PUSHDOWN queries are not supported
+query error db error: ERROR: EXPLAIN FILTER PUSHDOWN queries for this query type are not supported
+EXPLAIN FILTER PUSHDOWN FOR CREATE MATERIALIZED VIEW foo AS SELECT * FROM numbers where value > 10;
+
+# However, EXPLAIN FILTER PUSHDOWN FOR SELECT is now supported
+
+query TIIII
+EXPLAIN FILTER PUSHDOWN FOR SELECT * FROM numbers where value > 10;
+----
+materialize.public.numbers  0  0  0  0
+
+statement ok
+INSERT INTO numbers VALUES (1), (2), (3);
+
+# The next two queries may be slightly brittle, since they depend on part sizes.
```

Comment:
I'd guess that yeah that will be pretty brittle in practice. Could insert something like a 1024 byte string and then replace the `*` with something like `total_bytes > 1024 && total_bytes < 1024 * 10, selected_bytes, total_parts, selected_parts`, which would give you a result of `true 0 1 0`

Reply:
Unfortunately, I don't know a way to apply a function to the output of `explain` in an SLT test.

I think I can do something similar in `testdrive`? But I'm tempted to defer the port until either I have more similar tests to add or the brittleness here starts causing trouble.

Reply:
Oh, right! Ignore this